projects/fluid_simulation/main.cpp

```diff
@@ -119,6 +209,12 @@ void ProjApp::Render()
     // Update the simulation state.  This schedules new compute shaders to draw the next frame.
     mSim->Update();
 
+    // Draw Knobs window
+    if (GetSettings()->enableImGui) {
+        UpdateKnobs();
```

Comment:
If I decide to disable ImGUI, this will prevent the settings from taking effect.  Take the call to `UpdateKnobs` out of this `if()`?